Soccar [UI]/src/nl/socnet/message/SpawnCarMessage.java

```diff
@@ -0,0 +1,47 @@
+package nl.socnet.message;
+
+import nl.soccar.socnet.message.Message;
+import nl.soccar.socnet.message.MessageConstants;
+import nl.soccar.socnet.message.MessageEvent;
+import nl.socnet.message.handler.SpawnCarMessageHandler;
+
+/**
+ * @author PTS34A
+ */
+@MessageEvent(id = MessageConstants.SPAWN_CAR_MESSAGE_ID, handler = SpawnCarMessageHandler.class)
+public final class SpawnCarMessage extends Message {
+
+    private final int playerId;
+    private final float x;
+    private final float y;
+    private final float angle;
+
+    public SpawnCarMessage(int playerId, float x, float y, float angle) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)